package.json

```diff
@@ -41,74 +40,59 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@babel/cli": "^7.1.2",
-    "@babel/core": "^7.1.2",
-    "@babel/node": "^7.0.0",
-    "@babel/plugin-proposal-object-rest-spread": "^7.0.0",
-    "@babel/preset-env": "^7.1.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-jest": "^23.6.0",
-    "cross-env": "^5.2.0",
-    "es-check": "^4.0.0",
-    "eslint": "^5.7.0",
-    "eslint-config-jest-files": "^0.1.3",
-    "eslint-config-unicorn-camelcase": "^0.1.1",
-    "eslint-plugin-prettier": "^3.0.0",
-    "flux-standard-action": "^2.0.3",
-    "husky": "^1.1.2",
-    "jest": "^23.6.0",
-    "lint-staged": "^7.3.0",
-    "npm-run-all": "^4.1.3",
-    "prettier": "^1.14.3",
-    "rimraf": "^2.6.2",
-    "rollup": "^0.66.6",
-    "rollup-plugin-babel": "^4.0.3",
-    "rollup-plugin-commonjs": "^9.2.0",
-    "rollup-plugin-node-resolve": "^3.4.0",
-    "rollup-plugin-replace": "^2.1.0",
-    "rollup-plugin-terser": "^3.0.0",
-    "xo": "^0.23.0"
+    "@vitest/coverage-c8": "^0.26.2",
+    "eslint-config-unicorn-camelcase": "0.1.1",
+    "flux-standard-action": "2.1.2",
+    "husky": "8.0.2",
+    "lint-staged": "13.1.0",
+    "npm-run-all": "4.1.5",
+    "prettier": "2.8.1",
+    "vite": "4.0.3",
+    "vitest": "0.26.2",
+    "xo": "0.53.1"
   },
   "dependencies": {
-    "invariant": "^2.2.4",
-    "just-curry-it": "^3.1.0",
-    "loose-envify": "^1.4.0",
-    "reduce-reducers": "^0.4.3",
-    "to-camel-case": "^1.0.0"
+    "just-curry-it": "5.3.0",
+    "reduce-reducers": "1.0.4"
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
-  "browserify": {
-    "transform": [
-      "loose-envify"
+  "lint-staged": {
+    "*.{js,md}": [
+      "prettier --write",
+      "git add"
     ]
   },
   "xo": {
     "prettier": true,
     "space": true,
     "extends": [
-      "unicorn-camelcase",
-      "jest-files"
+      "unicorn-camelcase"
     ],
     "ignores": [
-      "rollup.config.js"
-    ]
+      "vite.config.js"
+    ],
+    "rules": {
+      "import/no-anonymous-default-export": "off",
+      "unicorn/no-array-reduce": "off",
+      "n/file-extension-in-import": "off",
+      "import/extensions": "off",
+      "unicorn/prevent-abbreviations": "off",
+      "unicorn/no-array-for-each": "off",
+      "unicorn/no-for-loop": "off",
+      "padding-line-between-statements": "off",
+      "unicorn/no-array-callback-reference": "off",
+      "n/prefer-global/process": "off",
+      "unicorn/prefer-export-from": "off",
+      "unicorn/prefer-array-flat": "off"
```

Comment:
I think for the future we should enable these rules and refactor the code to adhere to the linting rules.

Reply:
Yes totally agree, though some fight with prettier code formatting :D Might need to do some research to sort this out.